### DISCUSSION TITLE
GitHub Education Communicate using Markdown task autograding keeps failing

### DISCUSSION THREAD

**Discussion by azkadz** [community/community#114721] [GitHub Education]
### Hi everyone,

I've completed the tasks and I have added the necessary markdown prompts but the autograder keeps saying the tests have failed.
I'm not sure how to fix it, how to add another pull request to edit the index.md file or even to just completely start from scratch again.

https://github.com/Exp-Communicate-Using-Markdown-Cohort-1/series-communicate-using-markdown-azkadz

Please help, thank you very much in advance.

**Comment by Flavio13Lins** [comment]
Hi @azkadz .
Seems that it's a common issue, but it does not stop your learning process in GitHub Classromm.
Access https://education.github.com/pack/offers to see more tools of your interest and keep learning! 😊
